### ISSUE TITLE
CSS Advice

### ISSUE THREAD

**Issue by seattlecajun** [kyle-kelly/MMM-uber#2]
Hi there, kudos on this module! Seriously, it's super cool and very helpful. I was wondering if you know of any CSS advice to make the display a little cleaner.

I've removed the header, which helped, and set the top margins to -20 in both the MMM-uber and MMM-lyft module CSS files, but it's still not quite right.

Oh, and I was also wondering if I'm doing this right. I want to show info for two ride_type's each - UberX, UberXL, Lyft, and Lyft Line. Right now, I'm just calling up the modules twice and specifying a different ride_type with each in order to achieve this. 

Unrelated, but the Lyft module is showing "Lyft Line in 4 min + null Prime Time". I'm figuring it's just an issue with my communication through the Lyft API, but curious if you've come across that before.

Thanks again!

**Comment by kyle-kelly** [comment]
Thanks! Unfortunately I'm pretty shit at CSS so I'm about as useful as a StackOverflow search. However, I hacked my setup together a bit...below is a pic.

<img width="247" alt="screen shot 2018-01-12 at 10 25 12 am" src="https://user-images.githubusercontent.com/19912301/34881976-0ac7b65e-f783-11e7-9e6d-4489f3a8e4eb.png">

To achieve this you need one lyft module and two uber modules in the config. Then I changed the header for the lyft to module to `header: 'UBER & LYFT WAIT TIMES',`, omitted the headers for the two uber modules, and changed the uber CSS line to `margin-top: -40px;`.

As far as your lyft line issue, I never tested that ride type so you may have stumbled across a bug. I opened an issue [here](https://github.com/kyle-kelly/MMM-lyft/issues/1) where we can move that discussion.

Stay tuned because I think I'll release a new version that allows you to have multiple ride types per module.

**Comment by kyle-kelly** [comment]
I'm closing this because it isn't an issue as it relates to the code. I'll point you [here](https://forum.magicmirror.builders/topic/4295/mmm-uber-and-mmm-lyft) for further discussion.

**Comment by seattlecajun** [comment]
haha, yeah, same here. Regarding CSS and most things linux/raspberry pi related, for that matter, I know JUST enough to get myself into trouble.  Tinkering with this Magic Mirror setup for the last few weeks has been a really cool learning experiment though. :)

I saw your post on the MM forum originally, that's how I found your modules. I just wasn't sure about commenting on 4+ month old post. Some folks get really annoyed about that, but, if it's cool with you, it's cool with me!
